resolwe/flow/models/utils.py

```diff
@@ -154,6 +154,10 @@ def validate_data(data_pk, type_):
             field = _fields[name]
             type_ = _schema.get('type', "")
 
+            # Treat None as if the field is missing.
+            if not _schema.get('required', True) and field is None:
```

Comment:
Save `_schema.get('required', True)` to variable and also use it 10 lines above.